pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -2,11 +2,13 @@ import { useCallback, useState } from "react";
 import axios from "axios";
 import { useRouter } from "next/router";
 import { Signature } from "starknet";
-import { connect as starknetConnect, disconnect } from "starknetkit";
-
+import { connect as starknetConnect, disconnect as starknetDisconnect } from "starknetkit";
+// Import Stellar SDK (you'll need to add this dependency)
+import StellarSdk from "stellar-sdk";
```

Comment:
You import libraries but do not install them...
Or you forgot to push your package.json

Reply:
And you don't even use it